### ISSUE TITLE
how to set startup working_directory to home directory

### ISSUE THREAD

**Issue by jeffrson** [alacritty/alacritty#8278]
Hi,

there are couple of issues where it is discussed, why "~" or "$env:HOME" or "$HOME" doesn't work. But how is it supposed to work?
Static fixed string isn't quite useful if sharing configuration between users or operating systems.

Now as it seems, shell.program and shell.args don't work anymore, is there any other way?

BTW, for me on Windows, after installing Alacritty, the startup directory is "c:\windows\system32" :-(
"working_directory" as "." or "./" also don't help to get the home directory.

Best regards
Jeff

**Comment by chrisduerr** [comment]
Neither `~` nor `$HOME` are valid paths.

> Now as it seems, shell.program and shell.args don't work anymore, is there any other way?

They still work, they're just in a different location. See our documentation.

**Comment by jeffrson** [comment]
> Neither `~` nor `$HOME` are valid paths.

Yes, sure. I know why it doesn't work. I'd like to know how it could be done - independently of the OS.

> > Now as it seems, shell.program and shell.args don't work anymore, is there any other way?
> 
> They still work, they're just in a different location. See our documentation.

I had tried it of course. I just realized I missed the "[" ... "]" in args, which gives a weird error message:
`[0.002291900s] [ERROR] [alacritty_config_derive] Config error: shell: data did not match any variant of untagged enum Program`

Besides, that's ever been a workaround only.


> Besides, they still work from the old locations.

Well, yes, with an ugly message...
